src/imp/propagator_envoy.js

```diff
@@ -0,0 +1,187 @@
+import pb from 'protobufjs';
+import long from 'long';
+import _each from '../_each';
+import _leftpad from '../_leftpad';
+import SpanContextImp from './span_context_imp';
+import LightStepPropagator from './propagator_ls';
+
+const CARRIER_ENVOY_HEADER_KEY = 'x-ot-span-context';
+
+const BINARY_PROTO = {
```

Comment:
Not sure if this should be in here like this. I tried using a local file but it doesn't get generated with webpack. Any tips or recommendations for proto?

Reply:
> Not sure if this should be in here like this. I tried using a local file but it doesn't get generated with webpack. Any tips or recommendations for proto?

I think we should generate the [lightstep.proto](https://github.com/lightstep/lightstep-tracer-common/blob/master/lightstep.proto) in the same way that we generate the collector.proto. The resulting file would end up in the `src/imp/generated_proto` directory. See the [Makefile](https://github.com/lightstep/lightstep-tracer-javascript/blob/master/Makefile#L137-L142).

If we do that, we can probably avoid the dependency on `protobufjs` since we already use `google-protobuf`.